update_database_scripts/filterBetaUnits.py

```diff
@@ -0,0 +1,83 @@
+from sys import argv
+from os import path
+
+# Studies should have the beta unit for all associations that have the same trait/pValAnno/betaAnno/valType/studyID
+# combination. When they don't (either due to spelling errors in the beta unit column or otherwise), this script
+# removes those associations and prints out the number of associations removed as well as the studies affected.
```

Comment:
When you say it removes those associations, you mean that it removes the unique combination right?

Reply:
I mean it removes all associations from that combination. Is that correct?

Reply:
Yes :)